### PR TITLE
Use Yeoman for scaffolding

### PR DESCRIPTION
**Pull request by TheZoker**

### CODE REVIEW

File: generator-jsonforms/src/generators/app/index.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            {
+              name: 'Seed Project',
+              value: ProjectRepo.Seed
+            }
+          ],
+          when: () => {
+            if (this.project == null) {
+              return true;
+            }
+            /* if (!(Object.values(Project).includes(this.project))) {
```
Comment (edgarmueller) [inline generator-jsonforms/src/generators/app/index.ts]: Remove commented code

File: generator-jsonforms/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+This Yeoman Generator brings you all the functions of the tooling to your terminal. Within the terminal you can:
+
+* Create an example project
+* Create an seed project
+* Generate your basic UI Schema from your Schema
+
+## Get Started
+
+1. First you need to install yeoman via `npm i -g yo`
```
Comment (edgarmueller) [inline generator-jsonforms/README.md]: Please add a note here that this package has not yet been published.

File: generator-jsonforms/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+4. To run, just type `yo jsonforms`
+5. yeoman will ask you some questions (e.g. which project, which path etc.)
+
+If you want to avoid the interface, you can use the following parameters:
+
+### Project
+
+Choose the project you would like to scaffold
+
+Current available projects:
```
Comment (edgarmueller) [inline generator-jsonforms/README.md]: change 'project' to 'project seeds'

File: generator-jsonforms/src/generators/app/index.ts (hunk 1 above)
Comment (TheZoker) [reply generator-jsonforms/src/generators/app/index.ts]: This commented code refers to a question I posted in the technical-questions channel on slack.
This code should check if a value is inside a enum. But I wasn't able to get it working. Do you have any idea here?

**Pull request merged**